vaccine_feed_ingest/vial.py

```diff
@@ -84,7 +85,14 @@ def import_source_locations(
             body=encoded_ndjson.encode("utf-8"),
         )
 
-        rsp.raise_for_status()
+        if rsp.status != 200:
+            raise HTTPError(
+                f"/api/importSourceLocations?import_run_id={import_run_id}",
+                rsp.status,
+                rsp.data[:100],
+                dict(rsp.headers),
+                None,
+            )
```

Comment:
Thanks for setting the full details!

https://github.com/python/cpython/blob/7424105a02bbc209e032752636bae30ccb556402/Lib/urllib/error.py#L39-L44